jbmc/src/java_bytecode/java_types.h

```diff
@@ -260,7 +261,6 @@ bool is_reference_type(char t);
 // z  boolean
 // a  reference
 
-typet java_type_from_char(char t);
```

Comment:
Why move this?

Reply:
I actually moved the comment which is above it explaining which char represents what. That comment was previously connected to the deleted function `is_reference_type` so I thought instead of leaving it floating freely in the space I would move it to a function which actually has such a char as input argument.

Reply:
I guess I thought it more strongly related to `java_type_from_string`

Reply:
Uhh IMO if the comment is _below_ `java_type_from_char` I assumed it was related to `java_type_from_string`. I agree with @svorenova that this is more releated to `java_type_from_char` but would suggest putting it above and maybe as a doxygen comment. I'm not even sure what these letter relate to? The java bytecode signatures (as parsed by java_type_from_string) use upper case and `L` for reference no?

Reply:
It relates to `java_type_from_char`. Please turn it into doxygen.

Reply:
@thk123 They can apparently come from bytecode statements such as `aload`, for example `aaload`, `iaload`.